lib/ansible/modules/monitoring/icinga2_feature.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+
+Ansible module to manage Icinga2 feature state
+(c) 2016, Loic Blot <loic.blot@unix-experience.fr>
+
+Sponsored by Infopro Digital. http://www.infopro-digital.com/
+Sponsored by E.T.A.I. http://www.etai.fr/
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: icinga2_feature
+
+short_description: Manage Icinga2 feature
+description:
+    - Enable or disable an Icinga2 feature
+version_added: "2.3"
+author: "Loic Blot (@nerzhul)"
+options:
+    name:
+        description:
+            - Feature name to enable or disable
+        required: True
+    state:
+        description:
+            - Feature state to apply
+        required: false
+        choices: [ "present", "absent" ]
+        default: present
+
+extends_documentation_fragment: mysql
+'''
+EXAMPLES = '''
+# Enable ido-pgsql feature
```

Comment:
For examples please
1) Use ` - name: ...` rather than a comment
2) Use multi-line yaml `key: value`